### PR TITLE
travis.yml: temporarily allow beta task to fail

### PR DESCRIPTION
**Pull request by bgurney-rh**
Related: stratis-storage/project#168

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -12,6 +12,8 @@ addons:
 
 language: rust
 jobs:
+    allow_failures:
+          rust: beta
```
Comment (jbaublitz) [inline .travis.yml:16]: The standard for YAML is two spaces, not four. Given that this can introduce indentation bugs, we should stick with two spaces.

Comment (bgurney-rh) [reply .travis.yml]: I removed two spaces; does that work?

Comment (jbaublitz) [reply .travis.yml]: Oh sorry! That wasn't clear. I meant the indentation bumps should all be two spaces. The rest of the file has examples of this. Currently you have them at 4 spaces.

Comment (bgurney-rh) [reply .travis.yml]: okay; it took a few times to see what you meant, but hopefully I got everything.

File: .travis.yml [hunk 2]
```diff
@@ -12,37 +12,39 @@ addons:
...
-          env: TASK=release
+  allow_failures:
+    rust: beta
+  include:
+
+    # MANDATORY CHECKS USING CURRENT DEVELOPMENT TOOLCHAIN
+    - name: "format Rust source using current development toolchain"
+      rust: 1.42.0
+      before_script:
+          - rustup component add rustfmt
```
Comment (jbaublitz) [inline .travis.yml:23]: This line

File: .travis.yml [hunk 3]
```diff
@@ -12,37 +12,39 @@ addons:
...
+      env: TASK=fmt-travis
+
+    - name: "lint Rust source using current development toolchain"
+      rust: 1.42.0
+      before_script:
+          - rustup component add clippy
```
Comment (jbaublitz) [inline .travis.yml:29]: This line

File: .travis.yml (hunk 2 above)
Comment (bgurney-rh) [reply .travis.yml]: Resolved.

File: .travis.yml (hunk 3 above)
Comment (bgurney-rh) [reply .travis.yml]: Resolved.

**Comment by mulkieran** [comment]
Closing as obsoleted by subsequent changes and #71 .

**Pull request closed without merging**